### PR TITLE
Video consent copy

### PR DESCRIPTION
**Pull request by okaycj**
# Summary

This moves over video consent template five and the english translation file.

### CODE REVIEW

File: packages/record/templates/webcam-feed.mustache [hunk 1]
```diff
@@ -7,4 +7,4 @@
   height="{{height}}"
   style="display: inline-block"
 ></video>
-<img id="record-icon" src="{{{record_icon}}}">
\ No newline at end of file
+<img id="record-icon" src="{{&record_icon}}" />
```
Comment (becky-gilbert) [inline packages/record/templates/webcam-feed.mustache:10]: I wasn't familiar with this `{{&` mustache/handlebars syntax and had to look it up. Looks like it's used to return raw values? Is it also needed for the `play_icon` in the `playback-feed.mustache` template? Does `{{{` do the same thing?

File: packages/record/src/utils.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  );
+  Handlebars.registerHelper("exp-format", (context) => expFormat(context));
+};
+
+/**
+ * Initialize both i18next and Handlebars.
+ *
+ * @param trial - Yup
+ */
+export const init = (trial: TrialType<PluginInfo>) => {
```
Comment (becky-gilbert) [inline packages/record/src/utils.ts:66]: Picky request: could we change this `init` name to something a little more descriptive? Something like `initTemplatesAndTranslations` but not so long and clunky?

Comment (okaycj) [reply packages/record/src/utils.ts:66]: Yeah.  Let me see what I can come up with.

File: packages/record/src/consentVideo.ts [hunk 3]
```diff
@@ -1,40 +1,71 @@
...
+    research_rights_statement: { type: ParameterType.STRING, default: "" },
+    risk_statement: { type: ParameterType.STRING, default: "" },
+    voluntary_participation: { type: ParameterType.STRING, default: "" },
+    purpose_header: { type: ParameterType.STRING, default: "" },
+    procedures_header: { type: ParameterType.STRING, default: "" },
+    participation_header: { type: ParameterType.STRING, default: "" },
+    benefits_header: { type: ParameterType.STRING, default: "" },
+    risk_header: { type: ParameterType.STRING, default: "" },
+    summary_statement: { type: ParameterType.STRING, default: "" },
+    additional_segments: { type: ParameterType.COMPLEX },
```
Comment (becky-gilbert) [inline packages/record/src/consentVideo.ts:48]: For this `additional_segments` parameter, are we sticking with the EFP structure, i.e. an array of objects, where each object can have properties `title` and `text`? If so, I think you can make the parameter type a little more descriptive/restrictive, like this:
```js
additional_segments: {
  type: ParameterType.COMPLEX,
  array: true,
  nested: {
    title: {
      type: ParameterType.STRING, // or HTML_STRING?
      default: '',
    // etc.
```
Here's an example from another jsPsych plugin: https://github.com/jspsych/jsPsych/blob/a411b880690b97622ced0489e439fd9819dc5a08/packages/plugin-survey-multi-select/src/index.ts#L21-L52

File: packages/record/src/video_config.spec.ts [hunk 4]
```diff
@@ -56,13 +55,31 @@ let returnedDeviceLists: {
...
+    html
+      // Multiple whitespaces
+      .replace(/\s\s+/gm, " ")
+      // Whitespace and or slash before html element end
+      .replace(/\s*\/*>/gm, ">")
+      // equals empty string
+      .replace('=""', "")
+  );
+};
+
```
Comment (becky-gilbert) [inline packages/record/src/video_config.spec.ts:75]: Much better. Thanks for fixing this here too!

**Review by becky-gilbert — CHANGES_REQUESTED** [review]
Fantastic! A few questions/comments.
1. I think we need to add a Video Consent section to the `record` package docs (README). Hopefully you can just link to the EFP docs, if all of the parameters are still supported and work the same way.
2. I realize changesets/versioning doesn't actually matter yet, but just so that we're on the same page with how we want to do it: I _think_ that each changeset file should only list the packages that are affected by the change (feature/fix). So for instance, we'd have one that bumps the `record` package with: "Add video consent plugin". And the separate ones for switching to handlebars and adding translations, but again just for `record`? This PR does include changes to other packages, but those changes are just due to prettier line wrapping. So you could add a changeset with all packages for that, but IMO that sort of change doesn't warrant a changeset/version bump since it doesn't affect the code itself.

File: .changeset/gold-tips-melt.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+---
+"@lookit/lookit-initjspsych": patch
+"@lookit/surveys": patch
+"@lookit/record": patch
+"@lookit/style": patch
+"@lookit/data": patch
+---
+
+Replace MustacheJS with Handlebars
```
[the remark this answers is not in the document]
Comment (okaycj) [reply .changeset/gold-tips-melt.md:9]: If you like, I can add more text to explain why the other packages are included.

**Comment by okaycj** [comment]
@becky-gilbert 

> I think we need to add a Video Consent section to the record package docs (README). 

I'll get the documentation in a separate PR.  Thank you.

File: .changeset/gold-tips-melt.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (okaycj) [reply .changeset/gold-tips-melt.md]: Update the changeset to reflect changes more accurately.

File: .changeset/soft-trees-dream.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+---
+"@lookit/lookit-initjspsych": patch
+"@lookit/surveys": patch
+"@lookit/record": patch
+"@lookit/style": patch
+"@lookit/data": patch
+---
+
+Add remaining translations
```
[the remark this answers is not in the document]
Comment (okaycj) [reply .changeset/soft-trees-dream.md]: Update the changeset to reflect changes more accurately.

**Review by becky-gilbert — APPROVED** [review]
Looks great, thanks for the changes!

**Pull request merged**